corehq/apps/reports/views.py

```diff
@@ -2421,11 +2420,20 @@ def edit_form(request, domain, instance_id):
     instance = _get_location_safe_form(domain, request.couch_user, instance_id)
     assert instance.domain == domain
 
-    updates = {name: request.POST[name] for name in request.POST}
-    if FormProcessorInterface(domain).update_responses(instance, updates, request.couch_user.get_id):
-        messages.success(request, _('Question responses saved.'))
+    form_data, question_list_not_found = get_readable_data_for_submission(instance)
+    old_values, dummy = get_data_cleaning_data(form_data, instance)
+    updates = {name: request.POST[name] for name in request.POST
```

Comment:
This was pre-existing, but could this potentially conflict with the form in weird ways? Like if `request.POST.username` exists as some metadata in the request could that change a question called `username` in the form?

Reply:
It can, but I don't know how else to pass an arbitrary set of properties. Passing a dict value doesn't really work. We do this elsewhere (commcare settings) though that doesn't mean it's ideal.

Reply:
> Passing a dict value doesn't really work.

Do you mean passing a dictionary in a post parameter? Something that could be accessed like this:
```py
updates = json.loads(request.POST["updates"])
```

Reply:
What I had been doing was this in js:
```
$.ajax({
   ...
   data: {
      properties: {
        "a": 1,
        "b": 2,
        ...
      },
   },
})
```

which resulted in POST containing a value for each property, with a name like `properties[a]`. Passing in a JSON string instead and then using `json.loads` is indeed cleaner. Updated.